docs/enterprise-setup/self-managed/sso.md

```diff
@@ -42,12 +40,13 @@ Sign-out redirect URIs
 <your-airbyte-domain>/auth/realms/airbyte/broker/<app-integration-name>/endpoint/logout_response
 ```
 
-![Okta app integration name screenshot](./assets/docs/okta-login-redirect-uris.png)
+![Okta app integration name screenshot](../assets/okta-login-redirect-uris.png)
 
 _Example values_
 
 `<your-airbyte-domain>` should point to where your Airbyte instance will be available, including the http/https protocol.
 
+
 ## Deploying Airbyte Enterprise with Okta
```

Comment:
Seems we may be inconsistent with `Airbyte Self-Managed` vs `Airbyte Enterprise`.  Do we use both names or should be pick one?